pkg/genericapiserver/genericapiserver.go

```diff
@@ -248,12 +243,20 @@ func (s *GenericAPIServer) installGroupsDiscoveryHandler() {
 }
 
 func (s *GenericAPIServer) Run(options *options.ServerRunOptions) {
+	// This is currently separate from all other non-rest routes because
+	// the API group routes are created *after* initialization and you
+	// can't generate the swagger routes until after those are available.
+	// TODO eventually we should be able to factor this out to take place during initialization.
 	if s.enableSwaggerSupport {
-		s.InstallSwaggerAPI()
+		routes.SwaggerAPI{ExternalAddress: s.ExternalAddress}.Install(s.Mux, s.HandlerContainer)
 	}
 	if s.enableOpenAPISupport {
-		s.InstallOpenAPI()
+		routes.OpenAPI{
```

Comment:
... that's what I thought. Wikipedia: "The OpenAPI Specification (originally known as the Swagger Specification)".